docs/contributions/running-tests.md

```diff
@@ -0,0 +1,50 @@
+# Running Tests
+
+Tests are an important part of coding. Please refer to the following instructions when contributing to the repository.
+
+**Ensure you're on Node v10**
+If you need help updating, please see [NVM](https://github.com/creationix/nvm#installation). :heart: NVM.
+
+## Screenshot Tests
+
+Please refer to [screenshot tests](../screnshot-tests.md).
+
+## Unit Tests
```

Comment:
Add a section for running lint. Or include a note at the top that tells people to just run `npm test` for all tests